Ingress/controllers/gce/main.go

```diff
@@ -56,7 +60,7 @@ var (
 		printed to stdout and no changes are made to your cluster. This flag is for
 		testing.`)
 
-	clusterName = flags.String("gce-cluster-name", "default-cluster-name",
+	clusterName = flags.String("gce-cluster-name", "",
```

Comment:
do you mean what specifies the --gce-cluster-name argument? the pod in the test: 
https://github.com/kubernetes/kubernetes/pull/17614/files#diff-ec4b96325ef379121837e76bb579c833R257

It's actually just a uid used to tag cluster resoureces and I'm passing the test namespace through, so I suppose I should rename it to cluster-uid or something. I had initially name it gce-cluster-name because I though that's how people would use it.